### PR TITLE
Fix #1361-Text inside Edittext appears in selected mode.

### LINKED ISSUE #1361: Text inside the Edittext should be in selected mode when it gets focus.
**Actual Behaviour**

When an edittext which already has some text saved gets focus, the text does not appears in the selected mode.

**Expected Behaviour**

When an Edittext which already has some text saved gets focus, the text should appear in the selected mode.

**Steps to reproduce it**

Open up any edittext which already has some text saved like renaming an album operation.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sauravvishal8797**
Fix #1361 

Changes: Text inside Edittext appears in selected mode when the Edittext gets focus.

Screenshots for the change: 
screenshot_2017-10-19-22-21-53-281_org fossasia phimpme

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.4-all.zip
```
Comment (mohitmanuja) [inline gradle/wrapper/gradle-wrapper.properties]: Don't change gradle version.

**Comment by sauravvishal8797** [comment]
@mohitmanuja Done. Resolved the conflicts.

**Comment by mohitmanuja** [comment]
Resolve conflicts.

**Comment by sauravvishal8797** [comment]
@mohitmanuja Done.

**Pull request merged**